superset-frontend/src/visualizations/TimeTable/TimeTable.jsx

```diff
@@ -101,6 +100,7 @@ const defaultProps = {
 
 const TimeTableStyles = styled.div`
   height: ${props => props.height}px;
+  overflow: auto;
```

Comment:
You also need to remove the time-table class in line 328:

``<TimeTableStyles className={`time-table ${className}`} height={height}>`` 

`<TimeTableStyles className={className} height={height}>`

Reply:
@michael-s-molina this one as well is used by some Cypress tests